templates/ci/python-django.yml

```diff
@@ -0,0 +1,47 @@
+# Python Django
+# Test a Django app on multiple versions of Python.
+# https://aka.ms/yaml
+
+phases:
+
+# =================
+# Test a Django app
+# =================
+- phase: 'DjangoApp'
+  variables:
+    # relative path to your manage.py
+    projectPath: mysite/
+
+  queue:
+    name: 'Hosted Linux Preview'
+    parallel: 1
+    matrix:
+      Python35:
+        PYTHON_VERSION: '3.5'
+      Python36:
+        PYTHON_VERSION: '3.6'
+      Python37-dev:
+        PYTHON_VERSION: '>= 3.7.0a'
+  steps:
+
+  - task: UsePythonVersion@0
+    inputs:
+      versionSpec: '$(PYTHON_VERSION)'
+      architecture: 'x64'
+
+  - script: python -m pip install --upgrade pip setuptools wheel
```

Comment:
Would you consider upgrading setuptools and wheel a best practice?  If so, we should probably do it in python-package.yml as well.

Reply:
I typically `pip install -U pip setuptools` at the beginning of any new environment for Django. It mostly reduces logspam from pip complaining about being out of date. I think it's probably a good thing to upgrade these packages in the Python packaging task.

Updating `wheel` was a new one for me. I found it as I was trying to debug the "`pip` vs `python -m pip`" issue above. Having an up-to-date `wheel` package was mentioned on the `setuptools` project page; Django installation depends on `setuptools`. I'm not entirely clear that it's necessary. Thoughts?

Reply:
Personally I have this exact line in most of my automated builds. I'd stick with it.

Reply:
:+1: